### PR TITLE
Mobile navbar and resume link

### PR DESCRIPTION
**Pull request by t-creates**

### CODE REVIEW

File: tg-portfolio/components/layyout/Navbar.jsx [hunk 1]
```diff
@@ -48,24 +58,31 @@ const Navbar = () => {
...
         </div>
-        {/* Mobile Menu Button */}
-        <div className="md:hidden flex items-center">
-          button
+        {/* Mobile Menu */}
+        <div className="md:hidden flex items-center bg-green-800/90 w-full flex-row justify-between fixed">
+          <button onClick={toggleMenu} type="button"> <HiMenu className="h-8 w-8 text-white" /> </button>
+          <div style={{ display: open ? 'block' : 'none' }}>
+            <ul className="">
+              <div className="flex flex-auto flex-row mr-5 gap-1">
```
Comment (TidbitsJS) [inline tg-portfolio/components/layyout/Navbar.jsx:66]: You can apply the `div` styles to the `ul` element. No need to create an necessary `div`

File: tg-portfolio/components/layyout/Navbar.jsx [hunk 2]
```diff
@@ -48,24 +58,31 @@ const Navbar = () => {
...
+                {navLinks.map((navItem) => (
+                  <div key={navItem.id}>
```
Comment (TidbitsJS) [inline tg-portfolio/components/layyout/Navbar.jsx:68]: Again, it's not needed. You can add the key prop to the Link element

File: tg-portfolio/components/layyout/Navbar.jsx [hunk 3]
```diff
@@ -48,24 +58,31 @@ const Navbar = () => {
...
+                    <Link href={navItem.link}>
+                      <button type="button">
+                        <li
+                          key={navItem.listId}
```
Comment (TidbitsJS) [inline tg-portfolio/components/layyout/Navbar.jsx:72]: I don't think there is a need for this listId field. You can have a single unique ID per object and use that on the main element, i.e., over here, it is <Link />

**Review by TidbitsJS — APPROVED** [review]

File: tg-portfolio/components/layyout/Navbar.jsx (hunk 3 above)
Comment (t-creates) [reply tg-portfolio/components/layyout/Navbar.jsx:72]: Thanks!!

**Pull request merged**